config/dictionary_strings.rb

```diff
@@ -2031,6 +2031,12 @@
 _("Middleware Deployments")
 # TRANSLATORS: en.yml key: dictionary.table.middleware_deployments
 _("Middleware Deployments")
+# TRANSLATORS: en.yml key: dictionary.table.middleware_datasource
+_("Middleware Datasource")
+# TRANSLATORS: en.yml key: dictionary.table.middleware_datasource (plural form)
+_("Middleware Datasources")
+# TRANSLATORS: en.yml key: dictionary.table.middleware_datasources
+_("Middleware Datasources")
```

Comment:
@mzazrivec perhaps you are the right person who can explain us better the translation topic?